schemas/observation/master-prescribed-resource.js

```diff
@@ -5,7 +5,7 @@ var Joi = require( 'joi' );
 module.exports = {
 	'id'                     : Joi.string().guid().required().description( 'The resource id' ),
 	'resourceId'             : Joi.any().required().description( 'The id for the resource itself (e.g. contentId, groupId, fileId )' ),
-	'type'                   : Joi.any().required().valid( [ 1, 2, 3 ] ).description( 'The resource type (e.g. 1 for content, 2 for files, 3 for mc)' ),
+	'type'                   : Joi.any().required().valid( [ 1, 2, 3, 4 ] ).description( 'The resource type (e.g. 1 for content, 2 for files, 3 for mc, 4 for digital signature)' ),
```

Comment:
We argued this as well... In the end, we were **instructed** that Digital Signature be treated as a prescribed resource, you can check out the `EPIC ITEM: E-02559 MVP DIGITAL SIGNATURES`, all the stories under that epic were fitted to treat digital signature as PD.